cron/esCacheMaint.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# Copyright (c) 2014 Mozilla Corporation
+
+import json
+import logging
+import pytz
+import random
+import requests
+import sys
+import time
+from configlib import getConfig, OptionParser
+from datetime import datetime, date, timedelta
+from dateutil.parser import parse
+from elasticsearch import Elasticsearch
+logger = logging.getLogger(sys.argv[0])
```

Comment:
We also have a helper to create loggers, so you can simply "from utilities.logger import logger" and you no longer need this line, and the loggerTimeStamp and initLogger functions defined in this file.